src/App/App.module.css

```
@@ -2,38 +2,27 @@
   text-align: center;
 }
 
-.Logo {
-  height: 40vmin;
-  pointer-events: none;
-  border-radius: 50%;
+.Header {
```

why are you class names `PascalCase` ? it's no a big problem but we use  names like `.header` by convention

---

Now i use camel case selector naming in all app.
```suggestion
.app {
  text-align: center;
}

.header {
  color: var(--bg-dark-blue);
}
```